drivers/SmartThings/matter-switch/src/eve-energy/init.lua

```diff
@@ -0,0 +1,231 @@
+-------------------------------------------------------------------------------------
+-- Definitions
+-------------------------------------------------------------------------------------
+
+local capabilities = require "st.capabilities"
+local log = require "log"
+local clusters = require "st.matter.clusters"
+local cluster_base = require "st.matter.cluster_base"
+local utils = require "st.utils"
+
+local EVE_MANUFACTURER_ID = 0x130A
+local PRIVATE_CLUSTER_ID = 0x130AFC01
+
+local PRIVATE_ATTR_ID_WATT = 0x130A000A
+local PRIVATE_ATTR_ID_WATT_ACCUMULATED = 0x130A000B
+
+local LAST_REPORT_TIME = "LAST_REPORT_TIME"
+local TIMER_REPEAT = (1 * 60)    -- Run the timer each minute
+local REPORT_TIMEOUT = (10 * 60) -- Report the value each 10 minutes
+
+
+-------------------------------------------------------------------------------------
+-- Eve specifics
+-------------------------------------------------------------------------------------
+
+local function is_eve_energy_products(opts, driver, device)
+	if device.manufacturer_info.vendor_id == EVE_MANUFACTURER_ID then
```

Comment:
we use 2-width spaces instead of tabs

Reply:
Because of some unintended changesets in this PR, I created a new PR at https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/940

In this new PR, we use 2-width spaces instead of tabs